pkgs/development/python-modules/pygls/default.nix

```
@@ -49,9 +49,9 @@ buildPythonPackage rec {
   pythonImportsCheck = [ "pygls" ];
 
   meta = with lib; {
-    changelog = "https://github.com/openlawlibrary/pygls/blob/${src.rev}/CHANGELOG.md";
     description = "Pythonic generic implementation of the Language Server Protocol";
     homepage = "https://github.com/openlawlibrary/pygls";
+    changelog = "https://github.com/openlawlibrary/pygls/blob/v${src.rev}/CHANGELOG.md";
```

```suggestion
    changelog = "https://github.com/openlawlibrary/pygls/blob/${src.rev}/CHANGELOG.md";
```
`src.rev` already includes the prefix `v`.

---

oops, thanks for catching that!!